src/zabapgit_object_doct.prog.abap

```diff
@@ -88,7 +88,43 @@ CLASS lcl_object_doct IMPLEMENTATION.
 
   METHOD lif_object~jump.
 
-    lcx_exception=>raise( 'todo, jump DOCT' ).
+    DATA: ls_dokentry TYPE dokentry,
+          ls_bcdata   TYPE bdcdata,
+          lt_bcdata   TYPE STANDARD TABLE OF bdcdata.
+
+    ls_dokentry-username = sy-uname.
+    ls_dokentry-langu    = sy-langu.
+    ls_dokentry-class    = c_id.
+    MODIFY dokentry FROM ls_dokentry.
```

Comment:
this seems bad, is there any SAP standard function we can call? if not please add a comment